host_core/lib/host_core/actors/actor_module.ex

```diff
@@ -62,7 +63,7 @@ defmodule HostCore.Actors.ActorModule do
   end
 
   def halt(pid) do
-    GenServer.call(pid, :halt_and_cleanup)
+    if Process.alive?(pid), do: GenServer.call(pid, :halt_and_cleanup)
   end
```

Comment:
I wonder if this is a place that we can handle actors panicking, by knowing that it's not alive when we call `halt`

Just a thought, no change requested